LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 Utopia
+Copyright (c) 2022 Bradley Schofield
```

Comment:
I believe this needs to be Utopia itself. looking at the Appwrite license file for reference
https://github.com/appwrite/appwrite/blob/master/LICENSE